infrastructure/variables.tf

```diff
@@ -65,4 +65,8 @@ variable "aat_gateway" {
 
 variable "https_only" {
   default = "true"
+
+variable "common_tags" {
```

Comment:
Syntax error, curly brackets for `https_only` was moved to line 72

Reply:
Doh, fixed